tests/test_input_output_parameters.py

```diff
@@ -1,10 +1,11 @@
 import ansys.fluent.core as pyfluent
 from ansys.fluent.core import examples
+import pytest
 
 from ansys.fluent.parametric.parameters import InputParameters, OutputParameters
 
 
-def test_input_output_parameters():
+def test_input_output_parameters(monkeypatch: pytest.MonkeyPatch):
     case_filepath = examples.download_file("nozzle-2D-WB.cas.h5", "pyfluent/optislang")
```

Comment:
@mkundu1 

We have tried `case_filepath = examples.download_file("nozzle-2D-WB.cas.h5", "pyfluent/optislang", return_without_path=True)` still it is not able to find that file.

I have confirmed locally that `"nozzle-2D-WB.cas.h5"` exists in correct path.